Makefile

```diff
@@ -13,7 +13,7 @@ unit-test:
 .PHONY: int-test
 ## int-test: Runs the integration tests
 int-test:
-	docker-compose run --entrypoint=make dblab test
+	docker compose run --entrypoint=make dblab test
```

Comment:
I´d like to have this change in other PR rather than put it here on this feature branch.